include/souper/Extractor/ExprBuilder.h

```diff
@@ -0,0 +1,114 @@
+// Copyright 2014 The Souper Authors. All rights reserved.
```

Comment:
nit - since ExprBuilder.h is a newly-added file, we might want to make the copyright year start from 2018.